js/model.js

```diff
@@ -26,8 +27,7 @@ var mixerList = [
     {name: 'A-tail Quad',      model: 'quad_atail', image: 'atail_quad'},
     {name: 'Custom',           model: 'custom',     image: 'custom'},
     {name: 'Custom Airplane',  model: 'custom',     image: 'custom'},
-    {name: 'Custom Tricopter', model: 'custom',     image: 'custom'},
-    {name: 'Quad X 1234',      model: 'quad_x',     image: 'quad_x'}
+    {name: 'Custom Tricopter', model: 'custom',     image: 'custom'}
```

Comment:
This change breaks the functionality, since the list is indexed by https://github.com/betaflight/betaflight/blob/master/src/main/flight/mixer.h#L50-L78 .